src/App.jsx

```diff
@@ -1,20 +1,27 @@
-import { useState } from 'react'
-import './App.css'
+import { useState } from 'react';
+import './App.css';
 import { BrowserRouter, Route, Routes } from 'react-router-dom'
-import Header from './components/Header'
-import Login from './Login'
+import Header from './components/Header';
+import Login from './Login';
+import ArticlesList from './components/ArticlesList';
+import { UserProvider } from './contexts/UserContext';
 
 
 
 const App = () => {
+   
+  const [articles, setArticles] = useState([]);
```

Comment:
Does your article state need to be this high, or could it be lower? 🤔

Reply:
i have since moved it down.  I initially thought i would need it for topics but i didn't  thought